modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/client/strategy/AbstractExtensionsClientProcessorsStrategy.java

```diff
@@ -65,7 +67,7 @@ public AbstractExtensionsClientProcessorsStrategy(ExtensionManager extensionMana
     this.extensionManager = extensionManager;
     this.registry = registry;
     this.muleContext = muleContext;
-    this.policyManager = policyManager;
+    this.policyManager = policyManager != null ? policyManager : DEFAULT_POLICY_MANAGER;
```

Comment:
Can we either make the policyManager param an Optional, or force the client code to provide the NoOp?

Reply:
I created a ticket to completly rewrite ExtensionClient and policy manager shouldn't exist here anymore. I'd rather just wait for that ticket than to tamper with this API